x/bank/client/rest/tx.go

```diff
@@ -25,7 +26,23 @@ type SendReq struct {
 	Amount  sdk.Coins    `json:"amount" yaml:"amount"`
 }
 
+type stdTX struct { // nolint: deadcode unused
```

Comment:
This broke the integration @fedekunze. You don't need to define type aliases for request bodies -- we only do this hack for query responses where the `result` is a generic type.